### DISCUSSION TITLE
Filter IRC Post / String

### DISCUSSION THREAD

**Discussion by Friday13th87** [autobrr/autobrr#1229] [Q&A]
Is there a way to filter by words in the actual IRC string with the posting?
Under Advanced i already tried "Match releases" and "Match Release Tag" with and without regex, but it not filtering the IRC posting itself it seems.

**Comment by s0up4200** [comment]
Hi there. This is currently not possible. It was suggested as a feature earlier, but discarded due to disagreements. Maybe it will be reconsidered in the future.
